tc_driver/Cases.jl

```diff
@@ -377,22 +347,18 @@ function initialize_profiles(
     @. p_c = prof_p(z)
 end
 
-function surface_params(
-    case::LifeCycleTan2018,
-    surf_ref_thermo_state,
-    thermo_params,
-)
-    p_f_surf = TD.air_pressure(thermo_params, surf_ref_thermo_state)
-    ρ_f_surf = TD.air_density(thermo_params, surf_ref_thermo_state)
-    FT = eltype(p_f_surf)
+function surface_params(case::LifeCycleTan2018, thermo_params)
+    FT = eltype(thermo_params)
     zrough::FT = 1.0e-4 # not actually used, but initialized to reasonable value
+    psurface::FT = 1.015e5
     qsurface::FT = 22.45e-3 # kg/kg
-    θ_surface::FT = 299.1
+    θ_surface::FT = 299.1 # PAPER CHECK: Should we use Tsurface = 300.4 instead?
```

Comment:
This is a variation of Bomex case, so the same answer applies. For p=1015 hPa, T=300.4K and theta=299.1K is the same thing

Reply:
The case is flagged in the CI but the differences are indistinguishable. So it can be merged.